ceph.spec.in

```diff
@@ -846,7 +844,7 @@ mkdir -p %{_localstatedir}/run/ceph/
 %{_datadir}/ceph/id_dsa_drop.ceph.com
 %{_datadir}/ceph/id_dsa_drop.ceph.com.pub
 %dir %{_sysconfdir}/ceph/
-%dir %{_localstatedir}/log/ceph/
+%attr(3770,ceph,ceph) %dir %{_localstatedir}/log/ceph/
```

Comment:
FWIW: I suggest removing this line and modifying the other one, here. It will also resolve one of the warnings that rpm builds were having.